cost/kubecost/sizing/kubecost_resizing_recommendation.pt

```diff
@@ -124,6 +133,14 @@ datasource "ds_request_sizing" do
   end
 end
 
+script "js_get_request_from_iter_item", type: "javascript" do
```

Comment:
Can you speak to what this script is doing? I don't understand the value of a script that just returns whatever you feed into it.

Reply:
@XOmniverse - Hey!

I have a datasource that requires this script since I can't use `iter_item` directly for the `request` statement:
![image](https://github.com/flexera-public/policy_templates/assets/54189123/45ebdb6a-4d36-44a3-8f64-59bf0b5f249d)

So this script is just a helper for me to get the request object from the `iter_item` :)